bundles/core/src/main/java/com/adobe/cq/wcm/core/components/commons/link/Link.java

```diff
@@ -74,13 +72,24 @@ default String getURL() {
     }
 
     /**
-     * The processed URL, in case any extra processing can be done.
+     * The mapped URL, which supports mapping and vanity path.
```

Comment:
I'd go with: Returns the mapped URL ...

Reply:
The @return tag should make this clear. It's also not aligned with other interfaces like OEmbedClient for example.